test/bddtests/did_sidetree_steps.go

```diff
@@ -74,11 +74,6 @@ func (d *DIDSideSteps) checkErrorResp(errorMsg string) error {
 
 func (d *DIDSideSteps) checkSuccessResp(msg string) error {
 	if d.resp.ErrorMsg != "" {
-		documentHash, err := docutil.CalculateID(didDocNamespace, d.reqEncodedDIDDoc, sha2256)
```

Comment:
The title says 'remove panic'. but changes show other 2 lines removed. was this intentional ?

Reply:
yes